src/backend/Interner.zig

```diff
@@ -154,6 +168,13 @@ pub const Key = union(enum) {
                 128 => return .f128,
                 else => unreachable,
             },
+            .complex_ty => |bits| switch (bits) {
```

Comment:
I'm not sure how the type interner is used; should these be e.g. 64 bits for `cf32` since the type is 64 bits

Reply:
I think the way it's currently written is good since both parts are the same size. It might need to change when complex int support is added but that's a problem for then.